include/zephyr/drivers/fuel_gauge_sbs.h

```diff
@@ -0,0 +1,76 @@
+/*
+ * Copyright 2023 Google LLC
+ * Copyright 2023 Microsoft Corporation
```

Comment:
Directly copied definitions contributed by Microsoft, so also copied the copyright.